src/dls_bluesky_core/plans/__init__.py

```diff
@@ -0,0 +1,21 @@
+from .scanspec import scan
+from .wrapped import count
+
+"""
+This package is intended to hold MsgGenerator functions which as self-contained blocks:
+start runs, collect data, and close the runs. While they may be used as building blocks
+for larger nested plans, they are primarily intended to be run as-is, and any common
+functionality which may be useful to multiple plans extracted to stubs/.
+
+Plans must adhere to BlueAPI's requirements:
+- All arguments type hinted
+- Metadata dict named 'metadata'
```

Comment:
Do we definitely want this as a hard requirement? If so, there should be some clever test to make sure it's followed.

Reply:
Tests for the easily automated requirements added